### PR TITLE
Fixed documentation

### PR DESCRIPTION
**Pull request by td00**
changed example cfg paths to /var/pretalx/
added informations to deploy with ubuntu 16.04/debian 9

The change is for all peoply trying to deploy pretalx on a debian / ubuntu system.

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My change is listed in the CHANGELOG.rst if appropriate.

### CODE REVIEW

File: doc/administrator/installation_pip.rst [hunk 1]
```diff
@@ -69,8 +69,18 @@ Step 3: Package dependencies
...
+    # apt-get install git build-essential python3-virtualenv libssl-dev gettext libmysqlclient-dev
 
+
+For Ubuntu 16.04/Debian 9 you need the package "python3.6"::
+
+    # sudo add-apt-repository ppa:jonathonf/python3.6
+    # sudo apt-get update
+    # sudo apt-get install python3.6 python3.6-dev python3.6-venv
+    # wget https://bootstrap.pypa.io/get-pip.py
+    # sudo python3.6 get-pip.py
```
Comment (rixx) [inline doc/administrator/installation_pip.rst]: Please remove the "sudo" parts, as a leading ``#`` indicates superuser rights already.

**Comment by rixx** [comment]
Woop, thanks!

**Comment by td00** [comment]
@rixx removed the sudo's

**Comment by rixx** [comment]
Thanks!

**Pull request merged**